defaults/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+fail2ban_bantime: 3600
+fail2ban_findtime: 600
+fail2ban_maxretry: 3
+fail2ban_enabled: False
+fail2ban_backend: systemd
+fail2ban_logencoding: auto
+fail2ban_ignoreip:
+  - "127.0.0.1/8"
+fail2ban_jails:
+  - name: sshd
+    port: ssh
+    logpath: "%(sshd_log)s"
+    backend: systemd
+    enabled: True
+  - name: sshd-ddos
+    port: ssh
+    logpath: "%(sshd_log)s"
+    backend: systemd
+    enabled: True
+  - name: nginx-limit-req
```

Comment:
Are the jails pre-defined and only selected using the name? Can you reference a list of available jail configurations in the Readme?

Reply:
At the moment they aren't predefined. But I'm going to add a list of predefined jails later on. Preferably in another merge request.

Reply:
I plan on adding more predefined jails later on in a seperate pull request and also document them in the readme.
Right now the predefined jails are more for testing purposes.